packages/core/src/routes/resource.ts

```diff
@@ -3,6 +3,7 @@ import { Resources, Scopes } from '@logto/schemas';
 import { object, string } from 'zod';
 
 import { isTrue } from '#src/env-set/parameters.js';
+import RequestError from '#src/errors/RequestError/index.js';
```

Comment:
would you like to add some integration tests to cover this case?

Reply:
Good call, it do remind me about it